### ISSUE TITLE
Email settings have been made but it is not testing or sending. (Mail could not be sent. No additional error message provided. Check your mail settings and your app log.

### ISSUE THREAD

**Issue by mdedeoglu2** [grokability/snipe-it#14376]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Snipe-IT version:v6.3.1 build 12672 (g00cea3eb3)
PHP Version:8.1.2-1ubuntu2.14
Timezone:Europe/Istanbul
Mail From Address:IT [mdedeoglu@koyuncuoglu.com.tr](mailto:mdedeoglu@koyuncuoglu.com.tr)
Table Storage:cookieStorage
Software License:[AGPL3](https://www.gnu.org/licenses/agpl-3.0.en.html)
Laravel Version:8.83.27
Database Driver:mysql
Mail Reply-To Address:IT [mdedeoglu@koyuncuoglu.com.tr](mailto:mdedeoglu@koyuncuoglu.com.tr)

![env ayar](https://github.com/snipe/snipe-it/assets/162312737/988e4666-90f1-4d3d-8b89-4e1660d99216)

**I tried Snipe It with a fresh installation on Centos and Ubuntu, but it does not send e-mails.

There is no relevant data in /var/www/html/storage/logs/laravel.log.

What other logs can I check in the Ubuntu system?

![test summary](https://github.com/snipe/snipe-it/assets/162312737/8ba7d0c4-7d2a-4823-b920-cb5c5d43260f)

We cannot use Gmail because 'less secure app' support has been removed.

### Reproduction steps

1. **I tried Snipe It with a fresh installation on Centos and Ubuntu, but it does not send e-mails.

2. We cannot use Gmail because 'less secure app' support has been removed.

3.
...

### Expected behavior

We cannot use Gmail because 'less secure app' support has been removed.

### Screenshots

![env ayar](https://github.com/snipe/snipe-it/assets/162312737/ab58f7b8-11de-438c-834b-9f34607d07d8)
![test summary](https://github.com/snipe/snipe-it/assets/162312737/4c9c4a2c-38db-4b3a-9498-311f15318d3b)

### Snipe-IT Version

v6.3.1 build 12672 (g00cea3eb3)

### Operating System

Ubuntu 22

### Web Server

Apache

### PHP Version

PHP Version:8.1.2-1ubuntu2.14

### Error messages

```shell
There is no relevant data in /var/www/html/storage/logs/laravel.log.
```

**Comment by mdedeoglu2** [comment]
up

**Comment by mdedeoglu2** [comment]
The problem is in our mail server. I opened a new outlook.com email account and installed it with imap settings, no problem.

**Comment by snipe** [comment]
Glad to hear you're all sorted! In the future, if you've resolved your issue, please try to remember to close your ticket - it helps us out a lot. (We have to manage a lot of tickets.)

![tenant-tyvm](https://user-images.githubusercontent.com/197404/35019316-1bec03bc-fadb-11e7-92d3-0694b211e921.gif)

**Comment by koyuncuoglugithub** [comment]
[PHPMailer generates PHP Warning: stream_socket_enable_crypto(): Peer certificate did not match expected](https://stackoverflow.com/questions/30371910/phpmailer-generates-php-warning-stream-socket-enable-crypto-peer-certificate)


Then I realized that the problem persists. It requires authentication. I saw this in the Laravel logs. The company's e-mail account cannot be added. They suggest adding some things to the config/mail.php file, I tried but it didn't work.
I ask for your support, Miss Snipe It :)
